### ISSUE TITLE
Implement the solution for Winbind model

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#890]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/886
- Created at 2011-06-08 00:57:26 by [dpal](https://pagure.io/user/dpal)
- Closed as Invalid
- Assigned to nobody
---

This is the ticket to track the implementation of the design done in the scope of #885.

### Comments

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2011-06-08 14:51:44](https://pagure.io/SSSD/sssd/issue/886#233083)**

Fields changed

milestone: SSSD 1.6.0 => NEEDS_TRIAGE

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2011-06-09 15:18:18](https://pagure.io/SSSD/sssd/issue/886#233084)**

Fields changed

milestone: NEEDS_TRIAGE => SSSD 1.6.1

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2011-08-01 13:37:16](https://pagure.io/SSSD/sssd/issue/886#233085)**

This ticket depends on ticket #364 which is scheduled currently for 1.8.0. Moving to that bucket.

milestone: SSSD 1.6.1 => SSSD 1.8.0
rhbz:  =>

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2011-12-10 17:54:20](https://pagure.io/SSSD/sssd/issue/886#233086)**

Fields changed

milestone: SSSD 1.8.0 => NEEDS_TRIAGE

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2011-12-10 18:51:26](https://pagure.io/SSSD/sssd/issue/886#233087)**

Fields changed

milestone: NEEDS_TRIAGE => SSSD 1.8 AD Integration NEEDS TRIAGE

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2011-12-12 16:37:40](https://pagure.io/SSSD/sssd/issue/886#233088)**

Fields changed

milestone: SSSD 1.8 AD Integration NEEDS TRIAGE => SSSD Deferred

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2011-12-12 17:11:51](https://pagure.io/SSSD/sssd/issue/886#233089)**

Fields changed

owner: pzuna =>

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2012-01-19 03:21:33](https://pagure.io/SSSD/sssd/issue/886#233090)**

Fields changed

rhbz:  => 0

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2012-06-14 19:43:02](https://pagure.io/SSSD/sssd/issue/886#233091)**

Fields changed

blockedby:  => 
blocking:  => 
feature_milestone:  => 
resolution:  => wontfix
status: new => closed

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2017-02-24 14:47:00](https://pagure.io/SSSD/sssd/issue/886#233092)**

**Metadata Update from @dpal**:
- Issue set to the milestone: SSSD Patches welcome
